src/manifest/MsQuic.wprp

```diff
@@ -26,6 +26,25 @@
       </Stacks>
     </SystemProvider>
 
+    <SystemProvider Id="SP_SystemThreadExecutionDetail">
+      <Keywords>
+          <Keyword Value="ProcessThread"/>
+          <Keyword Value="Loader"/>
+          <Keyword Value="CSwitch"/>
+          <Keyword Value="ReadyThread"/>
+          <Keyword Value="SampledProfile"/>
+          <Keyword Value="DPC"/>
+          <Keyword Value="Interrupt"/>
+          <Keyword Value="IdleStates"/>
+      </Keywords>
+
+      <Stacks>
```

Comment:
```suggestion
      </Keywords>
      <Stacks>
```

Reply:
This is consistent with the block below.

Reply:
Otherwise I don't feel strongly one way or the other.